### ISSUE TITLE
feat: disableScreenshareSimulcast

### ISSUE THREAD

**Issue by psi-4ward** [jitsi/lib-jitsi-meet#1584]
We have a use case where it makes sense to have simulcast for Webcams enabled but not for screenshares. 
Are you open for a PR with an config option to disable simulcast for screenshares and setting it's bitrate? Like a follow up for unclear `capScreenshareBitrate`?

```
    desktopSharingFrameRate: {
         max: 30
    },
    desktopShareBitrate: 2e6,
    disableSimulcastScreenshare: true,
```

**Comment by jallamsetty1** [comment]
Yes, we would be open to a PR that provides the ability to control simulcast and bitrate for screenshare. `capScreenshareBitrate` was introduced as part of A/B testing effort when we started seeing issues with simulcast screenshare and it didn't make sense to continue using it.

**Comment by psi-4ward** [comment]
Nice.

Do you think there are any reasons to use simulcast for screenshare but not for video?
